pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -107,7 +107,8 @@ func NewCmdNewBuild(fullName string, f *clientcmd.Factory, in io.Reader, out io.
 	}
 
 	cmd.Flags().StringSliceVar(&config.SourceRepositories, "code", config.SourceRepositories, "Source code in the build configuration.")
-	cmd.Flags().StringSliceVarP(&config.ImageStreams, "image", "i", config.ImageStreams, "Name of an image stream to to use as a builder.")
+	cmd.Flags().StringSliceVarP(&config.ImageStreams, "image", "", config.ImageStreams, "Name of an image stream to to use as a builder. (deprecated)")
```

Comment:
There is a MarkDeprecated function from pflag that you can use. See https://github.com/openshift/origin/blob/62bcd6b8f5ee018035d1303f1db78f61f72136ef/pkg/cmd/cli/cmd/export.go#L77 for example